core/server/src/main/java/alluxio/web/WebInterfaceWorkerBlockInfoServlet.java

```diff
@@ -27,13 +27,17 @@
 import alluxio.worker.block.BlockWorker;
 import alluxio.worker.block.meta.BlockMeta;
 
+import org.apache.commons.lang3.tuple.ImmutablePair;
+
```

Comment:
I think now org and com imports are in the same session with new Alluxio code style.